doc/classes/ReflectionProbe.xml

```diff
@@ -51,6 +51,9 @@
 		<member name="origin_offset" type="Vector3" setter="set_origin_offset" getter="get_origin_offset" default="Vector3(0, 0, 0)">
 			Sets the origin offset to be used when this [ReflectionProbe] is in [member box_projection] mode. This can be set to a non-zero value to ensure a reflection fits a rectangle-shaped room, while reducing the number of objects that "get in the way" of the reflection.
 		</member>
+		<member name="priority" type="int" setter="set_priority" getter="get_priority" default="0">
+			Sets the priority of this probe. Only probes with the highest priority that touches a fragment is used. If more than one probe matches they are blended.
```

Comment:
It's a property but it reads like a setter. Sadly it's worth being consistent here, at least.
I would use something other than "touch". It doesn't feel right but I do not have the details to know what "touch" means here.
```suggestion
			Sets the priority of this probe, as a value between [code]0[/code] and [code]100[/code] (inclusive). If multiple probes touch a fragment, the one with the highest priority is used. If their priority is the same, the probes are blended.
```

Reply:
I think I'm showing my age, `touch` in this context is an old computer term for logic "touching" a bit of data, often referring to when you loop through something and access a related bit of data but not modifying it. I'm guessing it comes from the old days when we still had filing systems and you would browse through a bunch of cards "touching" the ones you examine.

Interestingly enough there is even a touch command on the command line (at least in linux/macos), where you can touch a file. It doesn't do anything with the file other than marking that file as accessed (usually by updating its last modified date even if nothing was modified).